.travis.yml

```diff
@@ -17,6 +17,9 @@ env:
 before_install:
 - curl -o- -L https://yarnpkg.com/install.sh | bash
 - export PATH="$HOME/.yarn/bin:$PATH"
+- curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+- export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
+- [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
```

Comment:
I think you might not need to curl nvm: https://discuss.circleci.com/t/nvm-does-not-change-node-version-on-machine/28973/14

Reply:
The link you posted is circleci and in this repo we're using travis. So I've gotta install it by myself :)

Reply:
oops, I forgot this repo uses travis